src/components/common/ProfilePanel.js

```diff
@@ -154,25 +154,27 @@ class ProfilePanel extends React.Component {
       );
     });
     return (
-      <ReactModal
+      <ImageSelector
         isOpen={this.state.showModal}
-        onRequestClose={this.handleCloseModal}
-        className="profile-image-modal"
-        overlayClassName="profile-image-overlay"
-        ariaHideApp={false}
+        closeModal={this.handleCloseModal}
+        icons={icons}
+        error={this.state.error}
+        maxWidth={325}
```

Comment:
Do you mind making this a constant somewhere? Just so we don't magic number ourselves too hard.